### DISCUSSION TITLE
color x based on value of x

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#53351] [Dashboards, Panels & Visualizations]
Greetings

In grafana is there a way to color the numeric values or background in the Stats viz based on another value in the data set. example

![image](https://user-images.githubusercontent.com/900951/183138872-042373ac-7505-4e27-acdb-ce66ea71ebe0.png)

So based on Text Column color Count stat?

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
